commit d40d50dab4b6a89f1e840364a47eeaf2ff7c6195 (refs/changes/43/3643/3)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-03-15T19:56:53+00:00 (1 year, 7 months ago)
    
    OS-6628 bhyve needs to support at least 32 vcpus
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/uts/i86pc/sys/vmm.h b/usr/src/uts/i86pc/sys/vmm.h
index cc87b8df30..757dc7ce7f 100644
--- a/usr/src/uts/i86pc/sys/vmm.h
+++ b/usr/src/uts/i86pc/sys/vmm.h
@@ -36,7 +36,7 @@
  * http://www.illumos.org/license/CDDL.
  *
  * Copyright 2015 Pluribus Networks Inc.
- * Copyright 2017 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 #ifndef _VMM_H_
@@ -418,7 +418,7 @@ void vm_copyout(struct vm *vm, int vcpuid, const void *kaddr,
 int vcpu_trace_exceptions(struct vm *vm, int vcpuid);
 #endif	/* KERNEL */
 
-#define	VM_MAXCPU	16			/* maximum virtual cpus */
+#define	VM_MAXCPU	32			/* maximum virtual cpus */
 
 /*
  * Identifiers for optional vmm capabilities
